packages/dropdowMenu/style.ts

```diff
@@ -14,6 +14,7 @@ export const menuList = css`
 
```

Comment:
the `overflow-y: scroll` looks weird in case you don't use an apple's trackpad and the scrollbar is rendered. that might be the issue @GeorgiSTodorov has screenshotted. https://github.com/dcos-labs/ui-kit/blob/9a6c61c4ce3fef8867ca6bff790c1c8ca577fe6f/packages/dropdowMenu/style.ts#L12

Reply:
I think you're right. I'll try and be smarter about when we apply this style